### PR TITLE
Allow specifying an optional runtime argument

### PR DESCRIPTION
**Pull request by jpadilla**
### Description

Allow specifying `runtime` on any request/command.

### CODE REVIEW

File: bin/_plugins/profile.js [hunk 1]
```diff
@@ -114,11 +117,25 @@ function sandboxFromArguments(args, options) {
...
+            return request;
+        }
+
         function onProfileLoaded(profile) {
             if (args.container) profile.container = args.container;
             if (args.url) profile.url = args.url;
             if (args.token) profile.token = args.token;
 
+            if (args.runtime) {
+                profile.onBeforeRequest = (profile.onBeforeRequest || []).concat(
```
Comment (johneke-auth0) [inline bin/_plugins/profile.js:134]: If `profile.onBeforeRequest` is defined here, is it guaranteed to always be an array? I couldn't figure that out

**Review by johneke-auth0 — COMMENTED** [review]
Looks good to me, but might be worth getting

File: bin/_plugins/profile.js (hunk 1 above)
Comment (jpadilla) [reply bin/_plugins/profile.js:134]: It should based on https://github.com/auth0/sandboxjs/blob/master/lib/sandbox.js#L43 but I'll add a guard against just to be safer 👍

**Review by johneke-auth0 — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**